packages/utils/src/query/types.ts

```diff
@@ -18,6 +18,7 @@ export type MetaData = {
   queryNodeVersion: string;
   startHeight?: number;
   rowCountEstimate: [TableEstimate];
+  deployments: Record<string, number>;
```

Comment:
Do we need to make this queryable?

Reply:
let's make it queryable, useful user track deployment history